roles/search/defaults/main.yml

```diff
@@ -1,5 +1,21 @@
 ---
 # defaults file for search
+data_folder: /opt/alfresco
+alfresco:
+  host: localhost
+  port: 80
+  port_ssl: 443
+
 search_flavor: alfresco-search-services  # set to alfresco-insight-engine for IE
-search_environment:  # This will add env vars at the end of the solr.in.sh file
-  SOLR_JAVA_MEM: -Xms1g -Xmx1g
+search_version: 2.0.5.1
```

Comment:
why we need a version as default?

Reply:
because `search_version` is defined  as a role argument in https://github.com/Alfresco/alfresco-ansible-deployment/pull/464/files#diff-bf95b68d6c46e569cab7ad9beeaf5657df593dc27d35a53d1a25ae1d488f4be4R83
We could make it a required argument but I feel like it make more sense to have the role install a predefined version (latest)